console.go

```diff
@@ -321,6 +321,8 @@ func consoleDefaultFormatLevel(noColor bool) Formatter {
 			default:
 				l = colorize("???", colorBold, noColor)
 			}
+		} else if i == nil {
+			l = ""
```

Comment:
I'm not certain of all the possible side-effects of this, but i would presume its safe enough if the input is nil, its probably preferable to not try to treat it as a string and just move on.